charts/agent/templates/_helpers.tpl

```diff
@@ -411,6 +411,12 @@ agent config to prevent a backend push from enabling them after installation.
             "secure_audit_streams") }}
             {{- $_ := set $secureConfig $secureFeature (dict "enabled" false) }}
         {{- end }}
+    {{ else if and $secureLightMode (semverCompare ">= 12.19.x" .Values.image.tag) }}
```

Comment:
@maratsal unfortunately we can't use the "semver*" functions since they'll make everything fail in case a non semantic version is provided. We should probably do something like we did in this PR https://github.com/sysdiglabs/charts/pull/1504

Reply:
U can add an helper like:
```
{{- define "agent.enableFalcoBaseline" }}
{{- if regexMatch "^v?([0-9]+)(\\.[0-9]+)?(\\.[0-9]+)?(-([0-9A-Za-z\\-]+(\\.[0-9A-Za-z\\-]+)*))?(\\+([0-9A-Za-z\\-]+(\\.[0-9A-Za-z\\-]+)*))?$" .Values.image.tag }}
{{- if semverCompare ">= 12.19.0-0" .Values.image.tag }}
{{- printf "true" -}}
{{- end }}
{{- end }}
{{- end }}
```
And then replace the `(semverCompare ">= 12.19.x" .Values.image.tag)` with `(eq (include "agent.enableFalcoBaseline" .) "true")`

Reply:
I see, let me add that logic. actually we probably can intro new function there that would include that regex in it, what do you think?

Reply:
added additional logic